### PR TITLE
Update the sign-up forms to be the new mailchimp lists

### LINKED ISSUE #88: Update the sign-up forms to be the new mailchimp mailing lists
The current forms are currently unofficially for SR2017, we've created new mailing lists for SR2018, so we should use them.

### PR DESCRIPTION
**Pull request by Adimote**
fixes #88

### CODE REVIEW

File: _includes/team-signup.html [hunk 1]
```diff
@@ -7,31 +9,72 @@
...
-      <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_1af17c2a3426de258620aae46_dd7d04139d" tabindex="-1" value=""></div>
-      <div class="clear">
-        <input class="button button-primary" type="submit" value="Register your interest" name="subscribe" id="mc-embedded-subscribe">
-        <span>or</span>
-        <a class="button" href="mailto:{{ site.emails.teams }}" target="_blank">Speak to someone</a>
+     <label for="mce-LNAME">Last Name </label>
+     <input type="text" value="" name="LNAME" class="" id="mce-LNAME">
       </div>
+<div class="mc-field-group size1of2">
+        <label for="mce-PHONE">Phone Number </label>
```
Comment (PeterJCLaw) [inline _includes/team-signup.html]: Do we really need to ask for phone number and postal address here?

As I understand it, this form is for people who are vaguely interested and would like to know more, rather than a full commitment to leading a team soon.

Given that, this level of detail feels like to put people off.

File: _includes/volunteer-signup.html [hunk 2]
```diff
@@ -1,23 +1,50 @@
...
+	<label for="mce-LNAME">Last Name </label>
+	<input type="text" value="" name="LNAME" class="required" id="mce-LNAME">
+</div>
+<div class="mc-field-group">
+	<label for="mce-MMERGE3">Location </label>
+	<input type="text" value="" name="MMERGE3" class="" id="mce-MMERGE3">
+</div>
+<div id="mergeRow-gdpr" class="mergeRow gdpr-mergeRow content__gdprBlock mc-field-group">
+    <div class="content__gdpr">
+        <label>Consent</label>
```
Comment (PeterJCLaw) [inline _includes/volunteer-signup.html]: Do we really need an explicit consent box?

They're giving us their information explicitly so that we can contact them about volunteering. We're not going to _market_ things to them (if we were then they would need to opt in to that), we're only going to tell them about things they're directly asking to be told about.

File: css/main.scss [hunk 3]
```diff
@@ -20,6 +20,5 @@ $baseurl: "{{ site.baseurl }}";
         "sponsors",
         "footer",
         "tags",
-        "splash",
```
Comment (PeterJCLaw) [inline css/main.scss]: This change seems unrelated.

File: volunteer.html [hunk 4]
```diff
@@ -70,8 +70,8 @@ <h3 class="text-center">Development</h3>
 <div class="panel">
   <div class="panel-container">
     <div class="panel-fill text-center row">
-      <div class="column l-6-12 l-offset-3-12">
         <a name="signup"></a><h2>Want to volunteer?</h2>
+      <div class="column m-8-12 m-offset-2-12 text-left">
```
Comment (PeterJCLaw) [inline volunteer.html]: Please can we fix the indentation of the previous line.

File: _includes/volunteer-signup.html (hunk 2 above)
Comment (Adimote) [reply _includes/volunteer-signup.html]: This form was generated using the mailchimp form generator, the consent checkbox can't be turned off, so I suspect their lawyers have looked into this and made this decision

File: css/main.scss (hunk 3 above)
Comment (Adimote) [reply css/main.scss]: oops, left over from a previous change

File: _includes/team-signup.html (hunk 1 above)
Comment (Adimote) [reply _includes/team-signup.html]: Currently all of these fields are set as 'required' in mailchimp, so I can't change this here.

@jimmythompson is there any reason why the phone number and address aren't optional?

Comment (Adimote) [reply _includes/team-signup.html]: Oops just looked into this, the phone number is actually optional, the point still stands for address however

File: _includes/volunteer-signup.html (hunk 2 above)
Comment (PeterJCLaw) [reply _includes/volunteer-signup.html]: Most likely MailChimp will be assuming that we're collecting the data to use it for marketing purposes. We're not. As I understand it, GDPR consent only applies to marketing messages.

File: volunteer.html (hunk 4 above)
Comment (Adimote) [reply volunteer.html]: fixed in 5a1219a220d4e74aa963c6b5d7bd4c34643cc123

File: _includes/volunteer-signup.html (hunk 2 above)
Comment (Adimote) [reply _includes/volunteer-signup.html]: I'm not certain I agree with you, but I'll remove it anyways because I trust your judgement and really don't want to spend another evening reading GDPR.

Comment (PeterJCLaw) [reply _includes/volunteer-signup.html]: Is there not an option in the MailChimp generator to say that we're not collecting _marketing_ emails?

Comment (PeterJCLaw) [reply _includes/volunteer-signup.html]: I think @prophile may have more GDPR understanding than I and thus may have an opinion here.

File: _includes/team-signup.html [hunk 5]
```diff
@@ -1,4 +1,6 @@
-<form action="//studentrobotics.us12.list-manage.com/subscribe/post?u=1af17c2a3426de258620aae46&amp;id=dd7d04139d"
+<!-- Begin MailChimp Signup Form -->
+<div id="mc_embed_signup">
+<form action="https://studentrobotics.us12.list-manage.com/subscribe/post?u=1af17c2a3426de258620aae46&amp;id=3ce424e7fa"
```
Comment (RealOrangeOne) [inline _includes/team-signup.html]: I believe the `&` is incorrectly escaped here?

**Comment by Adimote** [comment]
Oops looks like I merged in a bunch of changes in the latest push, will fix

**Comment by Adimote** [comment]
I've rebased it on top of master to fix conflicts, it should be good to go now

**Comment by PeterJCLaw** [comment]
@RealOrangeOne that's what local development environments are for, though I agree live linked previews are much nicer.
